src/v/config/configuration.cc

```diff
@@ -1468,6 +1468,12 @@ configuration::configuration()
       "cloud_storage_cluster_metadata_upload_interval_ms",
       "Time interval to wait between cluster metadata uploads.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      1h)
+  , cloud_storage_cluster_metadata_upload_timeout_ms(
+      *this,
+      "cloud_storage_cluster_metadata_upload_timeout_ms",
+      "Timeout for cluster metadata uploads.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       60s)
```

Comment:
assuming this controls how often we are uploading metadata, do we have some sort of napkin math for the amount of traffic this default would generate? e.g. i'd guess that offset metadata could be large in some cases?

Reply:
That would be `cloud_storage_cluster_metadata_upload_interval_ms`, which I should've called out is changed here to 1h. This here is an operational timeout (e.g. if it takes longer than a minute to upload offsets, fail the operation, perhaps retry, perhaps raise an alert, etc)